src/server/plugins/engine/pageControllers/SummaryPageController.ts

```diff
@@ -169,16 +170,19 @@ export class SummaryPageController extends PageController {
         return h.view('summary', summaryViewModel)
       }
 
-      const path = request.path
+      const { path, params } = request
       const isPreview = path.toLowerCase().startsWith(PREVIEW_PATH_PREFIX)
 
       // If not in preview mode, then send the submission email
       if (!isPreview) {
-        const emailAddress = this.model.def.outputEmail
+        // Get the form metadata using the `slug` param
+        const metadata = await getFormMetadata(params.slug)
+        const emailAddress =
+          metadata.notificationEmail ?? this.model.def.outputEmail
```

Comment:
Do you find yourself compromising with Prettier to reduce its weird new lines or just me?? 😆

```suggestion
        const { notificationEmail } = await getFormMetadata(params.slug)
        const emailAddress = notificationEmail ?? this.model.def.outputEmail
```

Reply:
Yes, but I've been trying to give it up as a bad habit but I'll indulge you this time :smile: cd027bfb1578789cc63c039e3a011e10e64c1613.